src/utils/helpers.js

```diff
@@ -25,3 +25,27 @@ export const setLocalStorage = (
   localStorage.setItem("activeSet", JSON.stringify(activeSet));
   localStorage.setItem("round", JSON.stringify(round));
 };
+
+export const chooseGroups = (groupSet) => {
+  let members = groupSet.values;
```

Comment:
It's not required to set the whole `groupSet` as argument, it's enough to bypass `groupSet.members`.
In addition, this function manipulate the argument in `members.splice(index, 1);`. In this case, it's better to create a copy.
Read more:
https://css-tricks.com/understanding-immutability-in-javascript/
https://reactjs.org/tutorial/tutorial.html#why-immutability-is-important

```suggestion
export const chooseGroups = (members) => {
  const membersClone = [...members]; 
```